std/math.d

```diff
@@ -4270,10 +4270,13 @@ struct IeeeFlags
 private:
     // The x87 FPU status register is 16 bits.
     // The Pentium SSE2 status register is 32 bits.
+    // PowerPC FPSCR is a 32-bit register.
+    // ARM FPSCR is a 32bit register
+    // SPARC FSR is a 32bit register (64 bits for SPARC 7 & 8, but high bits are uninteresting).
     uint flags;
+
     version (X86_Any)
```

Comment:
> [Microsoft is gonna try again with Windows on ARM, so CRuntime_Microsoft may not always imply x86(_64).]

That's fine, because it will probably use the same constants for both x86 and ARM. :-)